config/general.go

```diff
@@ -11,14 +16,28 @@ type GeneralConfig struct {
 	Mouse              bool           `toml:"mouse"`
 	Timestamps         bool           `toml:"timestamps"`
 	Identify           IdentifyConfig `toml:"identify"`
+	DownloadLocation   string         `toml:"attachment_download_location"`
 }
 
 func newGeneralConfig() GeneralConfig {
+
+	// We try to set the download folder location to the default Downloads folder
+	dlloc := "~/Downloads/"
+	if runtime.GOOS == "windows" {
+		h, _ := os.UserHomeDir()
+		dlloc = h + "\\Downloads\\"
+	} else if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
+		h, _ := os.UserHomeDir()
+		dlloc = h + "/Downloads/"
+	} else {
+		dlloc = os.TempDir() // Very lame fallback, I know
+	}
```

Comment:
- Extract this into a new function `func UserDownloadsDir() string`
- Remove the trailing slash at the end of the paths since this is later added using `filepath.Join()`; since, `os.TempDir()` returns path without a trailing slash, `os.Create()` call while downloading the file will return `/tmp<attachment.name>` instead of `/tmp/<attachment.name>`.